src/main.js

```diff
@@ -191,3 +197,57 @@ export const runPush = async () => {
         prevCommit = commitData.data.sha;
     }
 };
+
+export type __TestCommit = {
+    author: '__testAuthor',
+    comment_count: -1,
+    committer: '__testCommitter',
+    id: string,
+    message: string,
+    tree: '__TESTING__',
+    url: '__TESTING__',
+    verification: '__TESTING__',
+};
+
+export type __TestObject = {
+    context: __TestContext,
+    testNotified: string,
+};
+
+type __TestContext = {|
+    issue: {|owner: '__TESTING__', repo: '__TESTING__', number: -1|},
+    payload: {|
+        pull_request: {|base: {|ref: '__TESTING__'|}, user: {|login: '__testUser'|}|},
+        before: string,
+        after: string,
+        commits: Array<__TestCommit>,
+    |},
+    actor: '__testActor',
+|};
+
+/**
+ * @desc ONLY TO BE USED FOR TESTING. This is used to allow main.test.js to
+ * retrieve commits. This is necessary because main.test.js mocks a lot of the
+ * GitHub rest functions and "saves" commits into an object.
+ */
+export const __testGetCommit = async (commitSHA: string) => {
+    return await extraPermGithub.git.getCommit({
```

Comment:
Rather than making these methods and exporting them, it seems like it'd be better to export `extraPermGithub` and then you can do whatever calls you need on it to get the underlying data.